ops/ansible/playbooks-healthapt/templates/awslogs.conf.data-server.j2

```diff
@@ -135,60 +135,36 @@ log_stream_name = {instance_id}
 initial_position = start_of_file
 log_group_name = /var/log/secure
 
-[Jboss_Access_Log]
+[bfd_server_access_txt]
```

Comment:
You may need to rebase and make these modifications to the CW unified agent JSON configuration instead. Sorry!